tools/universe_builder.py

```diff
@@ -1,237 +1,43 @@
-#!/usr/bin/env python3
```

Comment:
Given that this has a main() function it may make sense to retain the executable bit + crunchbang so that it's directly executable. IIRC some of the bash tooling may call directly into `universe_builder.py`.